src/hyperion/parameters/rotation.py

```diff
@@ -44,6 +45,7 @@ class RotationScan(
         default=IspybExperimentType.ROTATION
     )
     transmission_frac: float
+    selected_aperture: AperturePositionGDANames
```

Comment:
David and I talked about this a bit yesterday - `GridCommon` uses `DiffractionExperiment`. Since the grid scans determine the aperture, why should the aperture bt an input parameter?

Reply:
Actually, your way works fine too, happy with that